store/tikv/util/misc.go

```diff
@@ -74,3 +76,60 @@ var SessionID = sessionIDCtxKey{}
 func SetSessionID(ctx context.Context, sessionID uint64) context.Context {
 	return context.WithValue(ctx, SessionID, sessionID)
 }
+
+const (
+	byteSizeGB = int64(1 << 30)
+	byteSizeMB = int64(1 << 20)
+	byteSizeKB = int64(1 << 10)
+	byteSizeBB = int64(1)
+)
+
+// FormatBytes uses to format bytes, this function will prune precision before format bytes.
+func FormatBytes(numBytes int64) string {
```

Comment:
Could we use this function in tidb because it is only used in log ... And it is just a proposal we can ignore it ..